packages/core/src/getPackageManifest.ts

```diff
@@ -0,0 +1,73 @@
+import child_process = require("child_process");
+let fs = require("fs-extra");
+let path = require("path");
+import { isNullOrUndefined } from "util";
+const xmlParser = require("xml2js").Parser({ explicitArray: false });
+
+export default async function getPackageManifest(projectDirectory, sourceDirectory) {
+    let mdapiDir: string = `${makefolderid(5)}_mdapi`;
```

Comment:
Can it also retun the folder where it is being convertd to? So that we could refactor the deploy source function as well